dir.props

```diff
@@ -114,7 +114,7 @@
   <!-- Dependency auto-upgrade configuration -->
   <ItemGroup>
     <DependencyBuildInfo Include="CoreFx">
-      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/$(DependencyBranch)</RawUrl>
+      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/dev/api</RawUrl>
```

Comment:
I think your right changes that do get committed might actually still merge although I'm still not 100% sure.

Your comment about using the devapi packages for a build or so in master is actually valid though and I do think that we will likely need to do that, so perhaps I'm just overthinking this :)

I think I better understand your suggestion now and I think it is reasonable.